Components/ApplePayDirect/Services/ApplePayButtonBuilder.php

```diff
@@ -50,20 +56,25 @@ class ApplePayButtonBuilder
 
 
     /**
+     * @param Account $accountService
      * @param Config $configMollie
      * @param \Shopware_Components_Config $configShopware
      * @param ApplePayPaymentMethod $applePayPaymentMethod
      * @param ApplePayDirectDisplayOptions $restrictionService
+     * @param \sAdmin|\Shopware_Proxies_sAdminProxy $sAdmin
+     * @param \sBasket|\Shopware_Proxies_sBasketProxy $sBasket
      */
-    public function __construct(Config $configMollie, \Shopware_Components_Config $configShopware, ApplePayPaymentMethod $applePayPaymentMethod, ApplePayDirectDisplayOptions $restrictionService)
+    public function __construct(Account $accountService, Config $configMollie, \Shopware_Components_Config $configShopware, ApplePayPaymentMethod $applePayPaymentMethod, ApplePayDirectDisplayOptions $restrictionService, $sAdmin, $sBasket)
     {
+        $this->accountService = $accountService;
         $this->configMollie = $configMollie;
         $this->configShopware = $configShopware;
         $this->restrictionService = $restrictionService;
+        $this->applePayPaymentMethod = $applePayPaymentMethod;
 
         # attention, modules does not exist in CLI
-        $this->sAdmin = Shopware()->Modules()->Admin();
-        $this->applePayPaymentMethod = $applePayPaymentMethod;
+        $this->sAdmin = $sAdmin;
```

Comment:
can you run the apple pay cli command and verify it does not crash, thanks

Reply:
There is no CLI command using this class. I didn't put the comment there, but I think it's meant as a warning in case there every would be a CLI command using the ApplePayButtonBuilder.